packages/nest/README.md

```diff
@@ -25,11 +25,45 @@
 [OpenFeature](https://openfeature.dev) is an open specification that provides a vendor-agnostic, community-driven API
 for feature flagging that works with your favorite feature flag management tool.
 
+<!-- x-hide-in-docs-end -->
+
 🧪 This SDK is experimental.
 
-#### Here's a basic example of how to use the OpenFeature NestJS API with `InMemoryProvider`.
+## Overview
+
+The OpenFeature NestJS SDK is a package that provides a NestJS wrapper for the [OpenFeature Server SDK]().
+
+It's main capabilities are:
+- Provide a NestJS global module to simplify OpenFeature configuration and usage within NestJS;
+- Supply custom parameter decorators (for controllers), that inject observables with feature flags resolution details;
+- Inject context for flag evaluation seamlessly using NestJS interceptors
+- Provide decorators for OpenFeature Client injection into NestJS services and controllers
+- TODO: map other features
```

Comment:
Do we want to remove this TODO?

Reply:
This is actually a reminder :sweat_smile:

@lukas-reining, is there anything else you think is worth adding to the docs at this point?
I'd say that from JSDocs + Readme it's pretty much all covered.

Reply:
I think this is fine, maybe something like "No need to access the global OF api for configuring OF" might be cool but is not required.

Reply:
Something like this @luizgribeiro: 

```suggestion
- Setting up logging, event handling, hooks and providers directly when registering the module
```

Reply:
I think the semicolons at the end of the points are not needed but not 100% sure :)